packages/cyberstorm/src/components/Toast/Toast.module.css

```diff
@@ -0,0 +1,181 @@
+}
+
+.contentWrapper {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  border-left: 3px solid var(--feature-color);
+  border-radius: var(--border-radius--4);
+  background-color: var(--background-color);
+
+  --background-color: var(--alert-info-bg-color);
+  --feature-color: var(--alert-info-accent-color);
+  --text-color: var(--alert-info-text-color);
+}
+
+.content {
+  display: flex;
+  gap: var(--space--16);
+  align-items: flex-start;
+  align-self: stretch;
+  width: 100%;
+  padding: 1rem 3.25rem calc(1rem - 0.125rem) 1.5rem;
+}
+
+.closeIconWrapper {
+  position: absolute;
+  top: 0.5rem;
+  right: 0.5rem;
+  background-color: transparent;
+}
+
+.closeIcon {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 1em;
+  height: 1em;
+  color: var(--color-text--tertiary);
+}
+
+.toastProgress {
+  width: 100%;
+  height: 0.125rem;
+  background-color: rgb(0 0 0 / 0.3);
+}
+
+.toastProgressBar {
+  height: 100%;
+  background-color: var(--feature-color);
+  animation: progress-bar var(--bar-timer) linear forwards;
+}
+
+/* Styles for Toast's text */
+.message {
+  display: flex;
+  flex: 1 0 0;
+  min-height: calc(1rem * var(--line-height--m));
+  color: var(--text-color);
+  font-weight: var(--font-weight-regular);
+  font-size: var(--font-size--m);
+  line-height: var(--line-height--m);
+}
+
+/* Styles for Toast's Icon */
+.icon {
+  width: var(--space--18);
+  height: var(--space--18);
+  color: var(--feature-color);
+}
+
+/* Different Toast variants */
+.toast__info {
+  --background-color: var(--alert-info-bg-color);
+  --feature-color: var(--alert-info-accent-color);
+  --text-color: var(--alert-info-text-color);
+}
+
+.toast__success {
+  --background-color: var(--alert-success-bg-color);
+  --feature-color: var(--alert-success-accent-color);
+  --text-color: var(--alert-success-text-color);
+}
+
+.toast__warning {
+  --background-color: var(--alert-warning-bg-color);
+  --feature-color: var(--alert-warning-accent-color);
+  --text-color: var(--alert-warning-text-color);
+}
+
+.toast__danger {
+  --background-color: var(--alert-danger-bg-color);
+  --feature-color: var(--alert-danger-accent-color);
+  --text-color: var(--alert-danger-text-color);
+}
+
+@keyframes progress-bar {
+  0% {
+    width: 100%;
+  }
+
+  100% {
+    width: 0%;
+  }
+}
+
+/* CSS for the Toast container */
+.toastContainer {
+  --viewport-padding: 25px;
+
+  position: fixed;
+  bottom: 0;
+  left: 0;
+  z-index: 2147483647;
+  display: flex;
+  flex-direction: column;
+  gap: 10px;
+  width: 390px;
+  max-width: 100vw;
+  margin: 0;
+  padding: var(--viewport-padding);
+  list-style: none;
+  outline: none;
+
+  --radix-toast-swipe-end-x: -25px;
+}
+
+.root[data-state="open"] {
+  animation: slide-in 600ms cubic-bezier(0.16, 1, 0.3, 1);
+}
+
+.root[data-state="closed"] {
+  animation: hide 200ms ease-in;
+}
+
+.root[data-swipe="move"] {
+  transform: translateX(var(--radix-toast-swipe-move-x));
```

Comment:
Where is this defined?

Reply:
That's a radix defined variable https://www.radix-ui.com/primitives/docs/components/toast#root